hyou/view.py

```diff
@@ -64,6 +64,18 @@ def _ensure_cells_fetched(self):
                 self._input_value_map.setdefault((index_row, index_col), value)
         self._cells_fetched = True
 
+    @api.retry_on_server_error
+    def clear(self):
+        params = {
```

Comment:
I suggest you add here as a docstring the clarification you wrote in the PR's description.

Reply:
Will do. Unfortunately the package doesn't have any docstrings so far, but we can try and add them as we go along.